server/public/js/domain/enemyPlayer.js

```diff
@@ -12,7 +12,7 @@ window.EnemyPlayer = class EnemyPlayer extends GenericPlayer {
     }
 
     doRender() {
-        this.game.otherPlayers.add(this.ship);
+        //this.game.otherPlayers.add(this.ship);
```

Comment:
Por que ahora no estamos ejecutando esto ?

Reply:
solo porque rompia el calculo de physics y no hacia nada mas (ni preguntes porque lo rompia.. solo por lo que vi reseteaba algunos valores )

Reply:
a oka, pero en otro lado se estaba agregando a la fisicas de `otherPlayers` no ?

Reply:
en GenericPlayer

Reply:
mmm pero como se supone que funcionan los choques ? 
en genericPlayer no creo que haya una referencia a `otherPlayers`

Reply:
que tipos de choques ? entre naves nunca hizo nada.. o del laser?

Reply:
Mala mia, me equivoque yo, por que el overlap se hace directamente contra `this.ship` en `removeSpawnProtection`.